app/response_svc.py

```diff
@@ -103,7 +103,7 @@ async def process_elasticsearch_results(self, operation, link):
 
             # Add result for new link
             new_result = Result(
-                id=str(new_link.id),
+                id=new_link.id,
```

Comment:
I was very curious about the `deepcopy` stuff going on above. That `apply_id(...)` call does _not_ do what I expected from the name, haha.